Pipeline/pipeline.py

```diff
@@ -62,7 +50,54 @@ def HTML_formater(df, name):
         f.write(html_string.format(table=df.to_html(classes="mystyle")))
 
 
-class pipeline:
+class Pipeline:
+    """
+
+    Parameters
+    ----------
+    df : pd.Dataframe
+
+    Examples
+    --------
+    >>> import pandas as pd
+    >>> import rossml as rsml
+
+    Importing and collecting data
+    >>> df = pd.read_csv('seal_fake.csv')
+    >>> df_val= pd.read_csv('xllaby_data-componentes.csv')
+
+    >>> df_val.fillna(df.mean)
+    >>> D = Pipeline(df)
+    >>> D.set_features(0, 20)
+    >>> D.set_labels(20, len(D.df.columns))
+    >>> D.feature_reduction(15)
+    >>> D.data_scaling(0.1, scalers=[RobustScaler(), RobustScaler()], scaling=True)
+    >>> D.build_Sequential_ANN(4, [50, 50, 50, 50])
+    >>> model, predictions = D.model_run(batch_size=300, epochs=1000)
+
+    Get the model configurations to change it afterwards
+    >>> # model.get_config()

```

Comment:
This model.get_config is a command to change neural networks parameters after adjusting it on pipeline without using it again. To further details, please check keras documentatation.

Reply:
But do we need to execute this command when running the example? 
Currently we have:
```python
>>> #model.get_config()
>>> D.model_history()
>>> D.metrics()
```
Should the code be:
```python
>>> model.get_config()
>>> D.model_history()
>>> D.metrics()
```
?

If the `model.get_config()` is not necessary to run the example I would remove this line.

Reply:
No,no...This model.get_config() is a advance tool. Is it used as ,say,a "last resort"

Reply:
One example of use is: imagine that you have just discovered that the first activation layer of your problem must be a 'tanh' and you have chosen a 'sigmoid'. To solve this situation, you use model.get_config to access your model and to change the activation function to the proper one. This could also be done with dropout, regularization and more.